config/defaults.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  taskcluster: {
+    queueBaseUrl: 'https://queue.taskcluster.net/v1',
+    authBaseUrl: 'https://auth.taskcluster.net/v1',
```

Comment:
I guess not.  They were defined in one of the packages that I used to refer to for setting up the configuration.